returnn/compile.py

```diff
@@ -82,9 +83,10 @@ def tasks(self):
     def run(self):
         if isinstance(self.returnn_config, tk.Path):
             returnn_config_path = self.returnn_config.get_path()
+            shutil.copy(returnn_config_path, self.out_returnn_config.get_path())
 
         elif isinstance(self.returnn_config, ReturnnConfig):
-            returnn_config_path = "returnn.config"
+            returnn_config_path = self.out_returnn_config.get_path()
             self.returnn_config.write(returnn_config_path)
 
         else:
```

Comment:
the else case would be `self.returnn_config` is a string. I think we also need a copy for this case (similar to line 86)

Reply:
Missing is that the file located at the "path as string" be copied to `self.out_returnn_config.get_path()`.

I mean independent of whether `self.returnn_config` is a `tk.Path` object or a `str` representing a path, the same action should be taken. And for the `tk.Path` case you do a copy so I thought in the `str` case a copy is missing.

Reply:
Sorry I looked at wrong code, will fix it now